src/libratbag-private.h

```diff
@@ -215,6 +224,16 @@ struct ratbag_resolution{
 	uint32_t capabilities;
 };
 
+struct ratbag_led {
+	struct ratbag_profile *profile;
+	enum ratbag_led_type type;
+	enum ratbag_led_mode mode;  /**< off; on; cycle; breath  */
+	struct ratbag_color color; /**< led color  */
```

Comment:
heh, at least these two comments shouldn't be needed. Mode is explained by the enum and color is, well color :)